emop-maven-plugin/src/main/java/edu/cornell/RppHandlerMojo.java

```diff
@@ -19,8 +19,8 @@
 @Mojo(name = "rpp-handler", requiresDependencyResolution = ResolutionScope.TEST)
 public class RppHandlerMojo extends MonitorMojo {
 
-    Set<String> criticalSpecsSet;
-    Set<String> backgroundSpecsSet;
+    static Set<String> criticalSpecsSet;
+    static Set<String> backgroundSpecsSet;
```

Comment:
Other than that question, lgtm

Reply:
This needs to be static because I want to use these values in `RppMojo`, but by the point at which `RppMojo` is invoked the instance of `RppHandlerMojo` I had earlier is already dead. By making these variables static, I can still access them even after the instance is dead. Hope this helps!